docs/metrics.asciidoc

```diff
@@ -503,3 +504,14 @@ Fields:
 
 * `${name}`: The value of `functionCounter.count()` (the delta when using `CountingMode.STEP`).
 --
+
+[float]
+[[metrics-only-mode]]
+=== Using the agent only for metrics collection
+
+There are cases where you would want to use the agent only to collect and ship metrics, without tracing any Java code.
+This may be useful, for example, when you already have an APM Server set up, and you want to ingest all metrics listed
+above from a JVM that runs something other than your application into Elasticsearch.
```

Comment:
[minor] I find that example a bit confusing. Especially this bit:

> runs something other than your application into Elasticsearch

3rd party applications may also be traced with the agent and you may just want metrics from a 1st party app.

Maybe just remove that sentence?

Reply:
OK